test/functional/deleteAll.ts

```diff
@@ -70,18 +70,41 @@ export default function deleteAllNotJustTestData(setupData, api) {
     deleteImages = images;
 
     if (deleteImages.length > 0) {
-      Promise.all(deleteImages.map((img) => {
-        const apiPath = getEndpointPath(img);
-        return api.del(setupData.apiUrl, apiPath);
-      }))
-        .then((responseBodies) => {
-          t.equal(
-            responseBodies.length,
-            deleteImages.length,
-            `resolved promises same length as delete images (${deleteImages.length})`);
-          t.end();
-        })
-        .catch(formatError);
+      deleteImages.forEach((image, idx, arr) => {
```

Comment:
Oops - multiple deletes still need s promise.all - otherwise will end the other test